app/controllers/application_controller.rb

```diff
@@ -1,4 +1,10 @@
 # frozen_string_literal: true
 
 class ApplicationController < ActionController::Base
+	def require_admin
```

Comment:
The indentations here look a little wide

Reply:
fixed